TokenRate.php

```diff
@@ -0,0 +1,46 @@
+<?
+
+namespace iFixit\TokenBucket;
+
+use \InvalidArgumentException;
+
+/**
+ * Defines a rate of tokens per second. Specify the tokens you want to
+ * allow for a given number of seconds.
+ */
+class TokenRate {
+   private $rate;
+
+   public function __construct($tokens, $seconds) {
+      if (!is_int($tokens)) {
+         throw new InvalidArgumentException("Tokens must be an int");
+      }
+      if (!is_numeric($seconds)) {
+         throw new InvalidArgumentException("Seconds must be an int");
+      }
+
+      $this->tokens = $tokens;
+      $this->seconds = $seconds;
```

Comment:
You're right, I meant for them to be private. If someone tried changing the seconds or tokens, it wouldn't update the rate, so yeah.